MAINTAINERS.md

```diff
@@ -0,0 +1,26 @@
+# OpenTelemetry Go Compile Instrumentation
+
+- [Overview](#overview)
+- [Current Maintainers](#current-maintainers)
+- [Approvers](#approvers)
+- [Emeritus](#emeritus)
+
+## Overview
+
+This document contains a list of maintainers in this repo. See [opensearch-project/.github/RESPONSIBILITIES.md](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#maintainer-responsibilities) that explains what the role of maintainer means, what maintainers do in this and other repos, and how they should be doing it. If you're interested in contributing, and becoming a maintainer, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## Current Maintainers
+
+| Maintainer         | GitHub ID                                         | Affiliation |
+| ------------------ | ------------------------------------------------- | ----------- |
+| Huxing Zhang       | [ralf0131](https://github.com/ralf0131)           | Alibaba     |
+| Przemyslaw Delewski| [pdelewski](https://github.com/pdelewski)         | Quesma      |
+| Romain Marcadier   | [RomainMuller](https://github.com/RomainMuller)   | Datadog     |
+| Kemal Akkoyun      | [kakkoyun](https://github.com/kakkoyun)           | Datadog     |
+| Liu Ziming         | [123liuziming](https://github.com/123liuziming)     | Alibaba     |
+
```

Comment:
We need to add a section for future approvers here.

Reply:
Future approvers are triage members and I will add that section.